themes/default/content/registry/packages/aws-native/installation-configuration.md

```diff
@@ -207,8 +207,7 @@ values:
     AWS_REGION: <YOUR_AWS_REGION>
   pulumiConfig: # exposes Pulumi config values to the Pulumi CLI
     project:environment: 'dev'
-    aws:roleArn: 'arn:aws:iam::058111598222:role/OrganizationAccountAccessRole'
-    aws:dynamodbEndpoint: 'dynamodb.us-east-2.amazonaws.com'
```

Comment:
This doesn't exist for aws-native. Conceptually this cannot even exist because aws-native is going through cloud control in order to communicate with aws services